packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -340,4 +342,90 @@ describe("BirthdayPicker", () => {
             expect(onChange).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe("labels", () => {
+        const translatedLabels: $Shape<Labels> = {
+            month: "Mes",
+            day: "Día",
+            year: "Año",
+            errorMessage: "Por favor ingrese una fecha valida.",
+        };
+
+        beforeEach(() => {
+            jest.useRealTimers();
+            DateMock.advanceTo(today);
+        });
+
+        it.each([defaultLabels.month, defaultLabels.day, defaultLabels.year])(
+            "renders the placeholder as %s",
```

Comment:
Nice! Thanks for including the value in the test output - makes for much more readable test output when we run in verbose mode. :)